app/views/course/surveys/index.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.surveys do
+  json.surveys do
+    json.array! @surveys, partial: 'survey.json', as: :survey
+  end
+  json.canCreate can?(:create, Course::Survey.new)
```

Comment:
Should be `Course::Survey.new(course: current_course)`, otherwise normal users won't be able to create the survey.